airbyte-cdk/python/bin/migrate_options_to_parameters.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+echo "Starting migration"
+
+[ -z "$ROOT_DIR" ] && exit 1
+
+CONNECTORS_DIR=$ROOT_DIR/airbyte-integrations/connectors
+CDK_DIR=$ROOT_DIR/airbyte-cdk/python/
```

Comment:
should we also migrate the connector builder server?

Reply:
Do you mean updating the starter manifest code in `default_api.py`? If so yes I agree that is a good suggestion

Reply:
yes, and the comments in the openapi spec https://github.com/airbytehq/airbyte/blob/master/airbyte-connector-builder-server/src/main/openapi/openapi.yaml#L74

Reply:
ah right adding!